indico/modules/events/registration/blueprint.py

```diff
@@ -77,6 +77,12 @@
                  'hide_registration', reglists.RHRegistrationHide, methods=('POST',))
 _bp.add_url_rule('/manage/registration/<int:reg_form_id>/registrations/<int:registration_id>/withdraw',
                  'manage_withdraw_registration', reglists.RHRegistrationManageWithdraw, methods=('POST',))
+_bp.add_url_rule('/manage/registration/<int:reg_form_id>/registrations/<int:registration_id>/remove_allow_modification',
+                 '_registration_remove_allow_modification',
```

Comment:
please don't start endpoint names with underscores, and use dashes instead of underscores in URLs

Reply:
Thanks, I updated the naming and removed the trailing underscores.